### DISCUSSION TITLE
NuxtJS / TailwindCSS - Custom fonts not loading in firefox browser only

### DISCUSSION THREAD

**Discussion by reachlinlen** [nuxt/nuxt#9697] [Questions (Nuxt 2)]
[https://lucid-williams-ebf8bf.netlify.app/](url)

My website uses NuxtJS / TailwindCSS. I'm facing two problems:

1. MacBook becomes slow when NuxtJS dev server is running
2. In Firefox, custom fonts are not downloading while Safari / Chrome have no issues

Tailwind CSS Font configuration can be seen at - [https://twitter.com/ungallenin/status/1428734141219053576?s=21](url)

**Comment by kissu** [comment]
Your clickable links are not aiming to the shown URL.
Check if you're using the latest Tailwind with JIT enabled, this way it will not dump all the classes into the DOM, only the used ones. You may maybe debug this directly by looking at the available CSS in your webpage.

As for Firefox, double-check the network tab to see what is happening with the fonts.
Maybe give a try to this solution: https://support.mozilla.org/en-US/questions/1286512

I also know that Firefox has the feature of disabling some default fonts loading of some sort, to basically help you find out if your custom font is working or not.

**Reply by reachlinlen** [reply]
Check if you're using the latest Tailwind with JIT enabled, this way it will not dump all the classes into the DOM, only the used ones. - Changed this config and now system doesnt hang.

**Reply by reachlinlen** [reply]
After adding font-weight and moving @font-face to tailwind config instead of keeping in global and importing lead to downloading fonts. But still fonts are not reflecting. Instead serif fonts are picked.
